metaflow/plugins/kfp/kfp_utils.py

```diff
@@ -59,276 +22,3 @@ def _get_kfp_logger():
 
 
 logger = _get_kfp_logger()
```

Comment:
Should the entire file be removed? Is `_get_kfp_logger` used outside this file?

Reply:
Ah I see this being used in the flow triggering flow PR. Maybe just move this logger over? There is nothing specific to KFP in this function. We just need to update the import path in `plugins/kfp/__init_.py` so that aip customers do not need to change their path.